.clabot

```diff
@@ -0,0 +1,6 @@
+{
+  "contributors": ["sgammon", "tyler-porras", "tsamartino", "samt", "RandyStevens", "noahdamiani", "mstan", "momentron", "cannabot", "KyleVandagriff"],
+  "message": "OpenCannabis requires contributors to sign a _Contributor License Agreement_, and we don\"t have one on file for you! In order for us to review and merge your code, please contact @OpenCannabis/editors to get yourself added.",
+  "label": "CLA: Ready",
+  "recheckComment": "clacheck"
```

Comment:
support for CLA bot. all Bloombox employees are auto-covered by the corp-to-corp CLA i've signed.